internal/rootcoord/root_coord.go

```diff
@@ -2519,19 +2610,25 @@ func (c *Core) checkSegmentLoadedLoop(ctx context.Context, taskID int64, colID i
 	defer ticker.Stop()
 	expireTicker := time.NewTicker(time.Duration(Params.RootCoordCfg.ImportSegmentStateWaitLimit*1000) * time.Millisecond)
 	defer expireTicker.Stop()
+	defer c.CallReleaseSegRefLock(ctx, segIDs)
```

Comment:
If an error occurred between locking and releasing the lock, can the lock be released successfully?

Reply:
locks will now be released when an import task completes, fails, or expires.